app/models/remote_record.rb

```diff
@@ -16,6 +16,10 @@ def self.bibdata?(source_metadata_identifier)
     PulMetadataServices::Client.bibdata?(source_metadata_identifier)
   end
 
+  def self.alma?(source_metadata_identifier)
+    bibdata?(source_metadata_identifier) && source_metadata_identifier.start_with?("99") && source_metadata_identifier.end_with?("6421")
```

Comment:
unfortunately there are voyager ids, e.g. `996421` but I'm not sure there's anything to be done about it.